tests/conftest.py

```diff
@@ -62,16 +61,8 @@ def pil_image(self) -> Image.Image:
     def for_hf(self) -> Image.Image:
```

Comment:
We should remove the `pixel_values` and `image_features` properties since they're no longer available via `download-images.sh`.

Reply:
good call, removed.